ark/utils/segmentation_utils.py

```diff
@@ -183,8 +183,7 @@ def concatenate_csv(base_dir, csv_files, column_name="fov", column_values=None):
     combined_data.to_csv(os.path.join(base_dir, "combined_data.csv"), index=False)
 
 
-def visualize_segmentation(segmentation_labels_xr, channel_data_xr,
-                           output_dir, chan_list=None, fovs=None, show=False):
+def save_segmentation_labels(segmentation_labels_xr, output_dir, fovs=None):
```

Comment:
We should add in a call to create_overlay and save the resulting RGB image to disk, without every displaying it